docs/content/users/basics/commands.md

```diff
@@ -977,6 +977,19 @@ ddev restart my-project my-other-project
 ddev restart --all
 ```
 
+## `self-upgrade`
+
+Give instructions on how to update or upgrade DDEV. Although the command doesn't actually do the upgrade (since it must be handled differently on different systems) it tries to figure out what you need to do and explain it to you. It must be executed in the context of a project.
+
+Example:
+
+```
+$ ddev self-upgrade
```

Comment:
Use `→  ` for shell prompt for consistency. (Not sure where it came from, I’ve just been following suit.)

```suggestion
→  ddev self-upgrade
```